redhat-developer/developers.redhat.com
# DEVELOPER-5533: Fixes for eloqua forms

Reworking eloqua embed to make it less wonky. Support overlabel visual style for embed forms.

### JIRA Issue Link
https://issues.jboss.org/projects/DEVELOPER/issues/DEVELOPER-5533

### Verification Process
* On any article node, add a "Content Call to Action" assembly
* Add the visual styles "inline" and "overlabel"
* In "content" add an H2 (title) and paragraph to match the mock. You can put whatever you want here tho.
* Add a new form with eloqua json like the following:
```
{
    "showAdditionalFields": [],
    "hideStandardFields": ['work phone', 'company', 'department', 'job role', 'country' ],
    "CustomQuestions": [],
    "offer_id": "70160000000Qf1SAAS",
    "language": "en",
    "FormTitle": "",
    "FormIntro": "",
    "NameOrder": "western",
    "FormCallToAction": "Sign Up",
    "ThanksTitle": "Thank you",
    "ThanksText": "You have been subscribed!",
    "ShowThanksButton": "true",
    "leadActivity": 1,
    "disableVisitorContactLookups": false
}
```

Save it all and verify the form appears correctly when viewing the article
